### ISSUE TITLE
"Like" feature for wars

### ISSUE THREAD

**Issue by jiakeliu** [gunnertech/thumbwar-ios-client#97]
Currently users can follow each other, but we need a way for users to keep track of wars that interest them. An intuitive way to accomplish this is to add a "Like" feature like the one in Instagram.

Here's the front end mock up, where the "Like" heart icon would appear for a war on the Home view. We need to keep track of the Like count of each post, and the user should have a way to look at the wars they "Liked" (likely in the Profile view).

![feed_v7](https://cloud.githubusercontent.com/assets/1039583/7485577/e8c6ced8-f34c-11e4-9502-e83896c19185.jpg)

**Comment by CodySwannGT** [comment]
Are we thinking this is going in MVP?

**Comment by jiakeliu** [comment]
I hope we can implement this in MVP. Is it realistic?

**Comment by CodySwannGT** [comment]
Not even a little bit if we're hoping to have the final MVP iteration end on May 24. It requires API and client work. Not to mention more definition - IE changing a like and like throttling.

**Comment by CodySwannGT** [comment]
Also - on the above point - we need a whole new view that would let a user filter wars that they've "Liked"

**Comment by jiakeliu** [comment]
That new view should be in the Profile screen, where it's currently displaying Won, Lost, Draw, and Active, we will just have a "Like" section.

**Comment by CodySwannGT** [comment]
Gotcha, regarding the views.

Think of it as this: It would take one iteration to build the API, and at least another iteration for the client to be able to leverage the API to make the writes and then probably a new iteration to implement the new view of reads. So, we're talking at least 2 iterations to bang this out and Adam's already jammed up with a full plate, so I would move this to future consideration and put it as top priority. Unless Alex/Bob feel this is important enough to add those iterations in.

**Comment by jiakeliu** [comment]
Agreed. I think there are many more high priority items out there right now. I'm moving this to future considerations.
